src/pages/lessons/[slug].tsx

```diff
@@ -109,6 +112,13 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
 
   if (!lesson) return null
 
+  React.useEffect(() => {
+    if (playerState.value === 'loading') {
+      // here we can do some consideration for the actual state.
+      send('PLAY')
```

Comment:
I just defaulted to play here, but you can do all sorts of fun algorithms here to figure out the state based on data we have if it is pro or whatever to pop a CTA

Reply:
Very cool. Should be a fun puzzle.